feature-libs/product-configurator/rulebased/components/group/configurator-group.component.spec.ts

```diff
@@ -480,22 +495,30 @@ describe('ConfiguratorGroupComponent', () => {
       const element = htmlElem.querySelector(
         '#cx-configurator--radioGroup_add--ATTRIBUTE_2_RADIOBUTTON_NUMERIC_ADDITIONAL_INPUT'
       );
-      CommonConfiguratorTestUtilsService.expectElementPresent(
-        expect,
-        element,
-        'cx-configurator-attribute-numeric-input-field'
-      );
+      if (!element) {
+        fail();
+      } else {
+        CommonConfiguratorTestUtilsService.expectElementPresent(
+          expect,
+          element,
+          'cx-configurator-attribute-numeric-input-field'
+        );
+      }
```

Comment:
This change and the following similar ones are not related to the feature. They are just fixes for the strict mode violations that i found in this spec.